### ISSUE TITLE
Deprioritize Grunt in the docs

### ISSUE THREAD

**Issue by addyosmani** [yeoman/yeoman.io#416]
We're at the point where a number of our most popular generators do not use Grunt. With this in mind, I think we should consider dropping or refactoring the messaging on our homepage to make it clear `yo` can be used to scaffold out a setup with any tooling choices.

Depending on how far we wanted to take it we could illustrate this point with examples.

E.g: 
Yo works great with Grunt. Check out generator-webapp.
Yo also works great with Gulp. Checkout generator-gulp-webapp.
It even works well with X, Y and Z which have nothing to do with either of these options.

If we decide to do this properly, we would also need to refactor the learning section of the site. Thoughts?

**Comment by arthurvr** [comment]
:+1: on all of that. Let me get out some PRs soon.

**Comment by addyosmani** [comment]
:+1:

**Comment by hemanth** [comment]
:+1:

**Comment by arthurvr** [comment]
Opened https://github.com/yeoman/yeoman.io/pull/417.

**Comment by arthurvr** [comment]
Other things that need to happen IMHO:

* [ ] Do we still want to have a big grunt logo on the front page?
* [ ] [This description of `yo`](https://github.com/yeoman/yeoman.io/blob/master/app/index.html#L59) seems coupled to grunt.
* [ ] Include some resources regarding gulp (and others) in the `learning/resources.html` page.
* [ ] The [contributing guidelines](https://github.com/yeoman/yeoman/blob/master/contributing.md#issue-submission) have some stuff specific to Gruntfiles.
* [ ] others(?)

For the rest I think we're quite fine, for the site at least. Will see if I have the time for some other PRs.

**Comment by addyosmani** [comment]
> Do we still want to have a big grunt logo on the front page?

Imo, if we want to be visual we could call out 1-3 build systems commonly used with Yo. Gulp and Grunt could be two, but we'd need to make sure we're citing them as examples rather than being all we support.

> This description of yo seems coupled to grunt.

Agreed! We should make it more generic.

> Include some resources regarding gulp (and others) in the learning/resources.html page.

:+1: 

> The contributing guidelines have some stuff specific to Gruntfiles.
 others(?)

We can similarly make this more generic I think.

**Comment by arthurvr** [comment]
Some of our [media](https://github.com/yeoman/media/blob/master/optimized/toolset.png) is also pretty tied to Grunt.

**Comment by SBoudrias** [comment]
@arthurvr are you still up with taking ownership of this issue?

If we need new medias, I can probably ask a designer somewhere for some help. Let me know what are your needs!

**Comment by arthurvr** [comment]
@SBoudrias Yeah, though somebody that can take care of the media and the homepage design would be welcome :)  

> Include some resources regarding gulp (and others) in the learning/resources.html page.

Anybody any suggestions what resources are really good?

**Comment by twixie** [comment]
> Anybody any suggestions what resources are really good?

The gulp site, and their repo contains a bunch of interesting information.

**Comment by amboy00** [comment]
Not sure if this is the right place to follow up, but will the site reflect that Yeoman uses Gulp over Grunt?  I noticed that the getting started generator is now on Gulr, even though the front page if this site states Grunt is an option.

**Comment by SBoudrias** [comment]
> will the site reflect that Yeoman uses Gulp over Grunt?

No. We're build tool agnostic.

**Comment by amboy00** [comment]
I understand that. However in the Getting Started section, the generator-webapp example no longer supports Grunt at all.  While it isn't misleading, technically, it does seem confusing.
